plugins/GSdx/GSRenderer.cpp

```diff
@@ -574,11 +574,13 @@ void GSRenderer::KeyEvent(GSKeyEventData* e)
 			return;
 		case VK_DELETE:
 			m_aa1 = !m_aa1;
-			printf("GSdx: (Software) aa1 is now %s.\n", m_aa1 ? "enabled" : "disabled");
+			theApp.SetConfig("aa1", m_aa1);
```

Comment:
usually, but I don't think there are any benefits in a temporary change. users couldn't find whether currently Mipmap is enabled (or) not after using multiple toggles without the following change. ( for those who don't use the console log). This change is currently important due to fact that users might take a look at the GUI for current status of Mipmap and AA1, outdated value might confuse them.